base/inference.jl

```diff
@@ -574,34 +618,43 @@ function tuple_tfunc(argtype::ANY)
     argtype
 end
 
-function builtin_tfunction(f::ANY, args::ANY, argtype::ANY)
-    isva = isvatuple(argtype)
```

Comment:
It just didn't look like it was really doing anything. `tuple` should still work with varargs, and the rest of these t-functions weren't doing much with isva.